src/fields.rs

```diff
@@ -899,180 +945,211 @@ The result of transforming `value as U`. Where `U` is wider than `T`, this
 zero-extends; where `U` is narrower, it truncates.
 **/
 fn resize<T, U>(value: T) -> U
```

Comment:
That's the only notable change actually. Big-endian should be tested, but I don't think there would be any problems.